easybuild/easyconfigs/v/VESTA/VESTA-3.5.8-gtk3.eb

```diff
@@ -0,0 +1,28 @@
+# Based on https://code.it4i.cz/sccs/easyconfigs-it4i/-/blob/it4i-karolina/v/VESTA/VESTA-3.5.7.eb
+
+easyblock = 'Binary'
```

Comment:
If this just extracting a tarball, then the `Tarball` easyblock is better suited

This does also require to make sure that the top-level install dir is added to `$PATH`, using:

```python
modextrapaths = {'PATH': ''}
```

(`Binary` does this automatically, `Tarball` does not)